focus-ios/Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -45,7 +45,9 @@
 		3AEC0B3E267DE30A007B7850 /* URIFixupTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3AEC0B3D267DE30A007B7850 /* URIFixupTests.swift */; };
 		4351A09928FCB8750087C1AF /* InfoPlist.strings in Resources */ = {isa = PBXBuildFile; fileRef = 4351A09728FCB8750087C1AF /* InfoPlist.strings */; };
 		4351A09C28FCB8750087C1AF /* Localizable.strings in Resources */ = {isa = PBXBuildFile; fileRef = 4351A09A28FCB8750087C1AF /* Localizable.strings */; };
-		45E8FFE72828DE4A0027A8F5 /* FocusAppServices in Frameworks */ = {isa = PBXBuildFile; productRef = 45E8FFE62828DE4A0027A8F5 /* FocusAppServices */; };
+		451B81BF2BA4DB9F00CF3C50 /* RustMozillaAppServices.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 451B81B92BA4DB9F00CF3C50 /* RustMozillaAppServices.framework */; };
+		451B81C02BA4DB9F00CF3C50 /* RustMozillaAppServices.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 451B81B92BA4DB9F00CF3C50 /* RustMozillaAppServices.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		451B81C92BA4DBB700CF3C50 /* FocusAppServices in Frameworks */ = {isa = PBXBuildFile; productRef = 451B81C82BA4DBB700CF3C50 /* FocusAppServices */; };
```

Comment:
Is this the actual name our could we use the same for what we use in Firefox? @tarikeshaq

Reply:
Also, did you need to add Glean as we do it for Firefox 
![Uploading Screenshot 2024-03-15 at 4.48.53 PM.png…]()

Reply:
eeeh the FocusAppServices name comes from rust-components-swift directly the RustMozillaAppServices name is the new framework (which is the same)

> Also, did you need to add Glean as we do it for Firefox

Nope, all I added was create the new framework and have it depend on the static lib, it's important to note that I'm out of my depth here with Xcode so I could have easily not done something right, but looks like tests are passing?